exporter-addon/pom.xml

```diff
@@ -5,14 +5,14 @@
 	<groupId>org.vaadin.addons</groupId>
 	<artifactId>exporter</artifactId>
 	<packaging>jar</packaging>
-	<version>1.0.0</version>
+	<version>1.0.1</version>
 	<name>Exporter</name>
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.6</maven.compiler.source>
-		<maven.compiler.target>1.6</maven.compiler.target>
-		<vaadin.version>7.6.6</vaadin.version>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Good point, will make change.  Strictly speaking, we don't need vaadin version 7.7.13 for my change.  Personally, I like keeping it, but do you think we should keep it at vaadin version 7.6.6, or does not not really matter?  Since I need to update Java version, thought I would ask.

Reply:
Vaadin version doesn't matter here, Vaadin dependency actually should be in **provided** scope.